src/locales/en/translation.json

```diff
@@ -46,34 +42,34 @@
     "numItems": "{{number}} items",
     "of": "of",
     "off": "Off",
-    "openAnnotationCompanionWindow": "Open annotation companion window",
-    "openCanvasNavigationCompanionWindow": "Open canvas navigation companion window",
-    "openInCompanionWindow": "Open in companion window",
-    "openInfoCompanionWindow": "Open information companion window",
-    "openWindows": "Open windows",
+    "openAnnotationCompanionWindow": "Annotations",
+    "openCanvasNavigationCompanionWindow": "Index",
+    "openInCompanionWindow": "Open in separate panel",
+    "openInfoCompanionWindow": "Information",
+    "openWindows": "Current open windows",
     "position": "Position",
-    "previewWindowTitle": "Mirador",
+    "previewWindowTitle": "{{title}}",
```

Comment:
Keeping this as an interpolated value for now in case we want to (or other languages would like to) provide some context around the title indicating it is being dragged.